webrtc/webapp/src/main/webapp/WEB-INF/conf/configuration.xml

```diff
@@ -0,0 +1,299 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>

```

Comment:
The content of this file should be moved to a new file WEB-INF/conf/webconferencing/configuration.xml and import tit from this file with war:/conf/webconferencing/configuration.xml (to allow customers to override it if necessary).